.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.9"
+services:
+  iree-dev:
+    build:
```

Comment:
Seems like we can prebuild these to save some startup time for devs? https://code.visualstudio.com/docs/devcontainers/containers#_prebuilding-dev-container-images

(Future work after landing this first PR)